core/templates/pages/admin-page/activities-tab/admin-prod-mode-activities-tab.component.html

```diff
@@ -0,0 +1,8 @@
+<mat-card class="oppia-page-card oppia-long-text oppia-long-text-card">
+The 'Activities' tab is not available in the production environment.
+</mat-card>
+<style>
+  .oppia-long-text-card {
+    max-width: 700px;
+  }
+</style>
```

Comment:
Is this style selector working? IIRC you need to make it more specific.

Reply:
I have attached screenshot of both the branches PTAL.